### DISCUSSION TITLE
C++ extension not working 1.10.5

### DISCUSSION THREAD

**Discussion by VarunArora14** [microsoft/vscode-cpptools#9443] [General]
I was doing my code in C++ but today it updated VS Code and C++ extension and it is not working, The Output tab says unable to spawn Intellisense and gives the below error.
**Problems Tab**
![image](https://user-images.githubusercontent.com/56734311/173525565-51106449-4df9-40a3-8c42-2773d811a98d.png)
**Output Tab for C++**
![image](https://user-images.githubusercontent.com/56734311/173525604-d884bf5b-7107-4c47-8b76-e19d655872b0.png)

Whenever I press ctrl+s it gives this on **Output** Tab  - 
![image](https://user-images.githubusercontent.com/56734311/173525829-0de01680-858e-4a05-a38b-61d1ca25b001.png)
Going back to 1.9.8 makes everything work as before but coming back to this breaks things again. Kindly fix this.

**Comment by VarunArora14** [comment]
![image](https://user-images.githubusercontent.com/56734311/173526837-d94dc339-79c3-405a-8973-01e7f33df343.png)
Running Clang.exe gives this where a cursor is just there and no error prompt

**Comment by sean-mcmanus** [comment]
1.10.x changed the mechanism we use to launch child processes, so it possible that change could be causing this issue for a subset of users. We're tracking this with https://github.com/microsoft/vscode-cpptools/issues/9431 .

Can you check if "C/C++: Run Code Analysis on Active File" also generates some sort of failure message, since it also involves creating a child process (I assume it would)...although it's possible it may silently fail depending on how we handle that error condition.

**Reply by VarunArora14** [reply]
It gives the message "Child exec failed u" in 1.10.5. Just to inform, 1.10.6 doesn't fix the problem as in #9431 with same errors mentioned above. Intellisense tries to find the path but is unable to do so.

**Reply by sean-mcmanus** [reply]
Yes, we've been able to repro the bug and have verified that our pending 1.10.7 will fix the issue (later today).
